### ISSUE TITLE
High Quality Image Preview over SSH works on broot version 1.33, but breaks on version 1.34+ 

### ISSUE THREAD

**Issue by i-api** [Canop/broot#910]

**Comment by i-api** [comment]
Leaving a comment so i get pinged later

**Comment by i-api** [comment]
https://github.com/Canop/broot/issues/505

**Comment by Canop** [comment]
Did you test with 1.40 ?

**Comment by i-api** [comment]
> Did you test with 1.40 ?

Yes I did -  High Quality Imagine Preview over SSH also breaks on 1.40 
<img width="833" alt="Screenshot 2024-07-21 at 4 46 54 PM" src="https://github.com/user-attachments/assets/7c1383ae-8f39-4cfb-890e-dd6bd819b121">
<img width="1096" alt="Screenshot 2024-07-21 at 4 47 04 PM" src="https://github.com/user-attachments/assets/7b6b0f2e-e5a5-484f-afb1-e628ac2a6e99">

**Comment by joihn** [comment]
same issue here,
img preview does not work over ssh, 
however (as a sanity check), `kitten icat myimg.png` does work over ssh

broot 1.44.2
kitty 0.37.0 
kitten 0.37.0

**Comment by Canop** [comment]
@joihn Did you set

    kitty_graphics_transmission: chunks

?

**Comment by joihn** [comment]
thanks, it does work after setting `kitty_graphics_transmission: chunks` :smiley: 
Thanks again for this awsome tool

**Comment by edrozenberg** [comment]
Similar issue here:
- High-res image preview does not work over ssh w/ MacOS Ghostty (client) ssh to Slackware Linux (server).

Using:
- broot `1.44.7` on Slackware Linux
- Ghostty `1.1.2` on MacOS 15.3.1 (24D70)
- SSH client: `OpenSSH_9.8p1, LibreSSL 3.3.6` (MacOS)
- SSH server: `openssh-9.9p2` (Slackware Linux)

Already set `kitty_graphics_transmission: chunks` in `conf.hjson`.
Using Ghostty directly on the Linux server on Xfce window manager works fine to show high-res images in broot.

**Comment by Canop** [comment]
Over ssh, `kitty_graphics_transmission: chunks` is needed (temp files aren't shared with the remote host). Since 1.50, Ghostty is detected through `TERM=xterm-ghostty`, and the `kitty_graphics_display` option (`unicode` works through tmux and ssh) covers the cases where auto-detection fails; see https://dystroy.org/broot/conf_file/#kitty-graphics-display. Closing; please open a new issue with your terminal and the debug log if it still fails on a current version.
